sycl/test/include_deps/sycl_detail_core.hpp.cpp

```diff
@@ -143,6 +143,19 @@
 // CHECK-NEXT: detail/spirv.hpp
 // CHECK-NEXT: ext/oneapi/experimental/non_uniform_groups.hpp
 // CHECK-NEXT: ext/oneapi/sub_group_mask.hpp
+// CHECK-NEXT: builtins.hpp
```

Comment:
Normally we would try to avoid growing this, but this change is a bugfix. The proper future change is to move the entire "sub_group_mask" extension out of `core.hpp`.